src/site/stages/build/drupal/graphql/page.graphql.js

```diff
@@ -34,6 +34,14 @@ module.exports = `
     title
     fieldIntroText
     fieldDescription
+    fieldFeaturedContent {
+      entity {
+        entityType
+        entityBundle
+        ${WYSIWYG}
```

Comment:
Is it worth adding the other items from `fieldContentBlock` here?

Reply:
as far as i know... no.
these can only be these two types, according to the drupal settings. i can double check with @beeyayjay 

on staging:
![screen shot 2019-02-12 at 2 19 15 pm](https://user-images.githubusercontent.com/19178435/52671942-3ee8d680-2ed1-11e9-97b0-e754fa28b52f.png)
the rest are unchecked as well

Reply:
@jbalboni @beeyayjay said that is correct - it can only be wysiwyg or q & a